bm25s/tokenization.py

```diff
@@ -634,5 +748,4 @@ def tokenize(
             )
         ):
             corpus_ids[i] = [reverse_dict[token_id] for token_id in token_ids]
-
-        return corpus_ids
+        return CorpusIDsList(corpus_ids)
```

Comment:
This is not a good idea, because it changes the return signature and would thus create a breaking change. I also do not see the benefit of abstracting the returned object with another class. Can you revert it back to the original returned object and remove `CorpusIDsList`?